lib/decidim/time_tracker/engine.rb

```diff
@@ -16,6 +16,11 @@ class Engine < ::Rails::Engine
           resources :activities do
             post :start, controller: "time_events"
             post :stop, controller: "time_events"
+
+            resource :join, controller: "activities_questionnaire"
```

Comment:
I have named "join" the resource used to answer the questionnaire, but I still don't manage to know if I should limit the endpoints using "only" as well as how to define the correct endpoints used to answer the questionnaires